0.4/schemas/image-label.schema

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://ngff.openmicroscopy.org/0.4/schemas/image-label.schema",
+  "title": "OME-NGFF well schema",
+  "description": "JSON from OME-NGFF .zattrs",
+  "type": "object",
+  "properties": {
+    "image-label": {
+      "type": "object",
+      "properties": {
+        "colors": {
+          "description": "The colors for this label image",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "label-value": {
+                "description": "The value of the label",
+                "type": "number"
+              },
+              "rgba": {
+                "description": "The color stored as an RGBA unsigned-int 4-tuple",
+                "type": "array",
+                "items": {
+                  "type": "integer",
+                  "minimum": 0,
+                  "maximum": 255
+                },
+                "minItems": 4,
+                "maxItems": 4
+              }
+            },
+            "required": [
+              "label-value"
```

Comment:
Is it not required to have `rgba` in the colors object? If you don't have a color for a particular label, you probably shouldn't have a `color` object for that value?

Reply:
That was my initial thought but the existing specification says `Additionally, the "rgba" key MAY be present`

So making it required would be a breaking change